common/landed_titles/01_landed_titles.txt

```diff
@@ -10866,7 +10866,7 @@ e_great_barrens = {
 					color2 = { 255 255 255 }
```

Comment:
I would make these isles pretty too.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
Resolved in e0982e4.